src/main/java/jsr223/nativeshell/shell/Shell.java

```diff
@@ -30,6 +30,8 @@ public ProcessBuilder createProcess(File commandAsFile) throws ScriptException {
                 } else if (!interpreter.canExecute()){
                     throw new ScriptException(String.format("Interpreter '%s' cannot be executed.", pathToInterpreter));
                 }
+            } else if (shebangLine.toLowerCase().contains("rexx")) {
+                // nothing to do
```

Comment:
why not
```
else if (!shebangLine.toLowerCase().contains("rexx")) {
     throw new ScriptException("Incorrect shebang notation: " + shebangLine);
}
```

Reply:
I thought about it but I think negation is not that easy to read